### PR TITLE
Fix issue 295: remove github URLs from the parent POM

### PR DESCRIPTION
**Pull request by sgilda**
Removed obsolete github URLs from POM. Added hbernate3 as issue is resolved.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -24,8 +24,7 @@
     <packaging>pom</packaging>
     <name>JBoss AS Quickstarts Parent</name>
     <description>JBoss AS Quickstarts Parent</description>
-    <url>http://github.com/jbossas/quickstart</url>
```
Comment (pmuir) [inline pom.xml:27]: Actually, why don't we provide http://jboss.org/jdf/quickstarts/get-started as the URL here?

**Comment by pmuir** [comment]
Looks good.

**Comment by sgilda** [comment]
Excellent idea to use the jdf URL. Modified and squashed.

**Comment by pmuir** [comment]
Merged

**Pull request closed without merging**